src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/GrantMandatoryQuestionsController.java

```diff
@@ -0,0 +1,72 @@
+package gov.cabinetoffice.gap.adminbackend.controllers;
+
+import gov.cabinetoffice.gap.adminbackend.services.GrantMandatoryQuestionService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.ContentDisposition;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.*;
+
+import static gov.cabinetoffice.gap.adminbackend.controllers.SubmissionsController.EXPORT_CONTENT_TYPE;
+
+@Log4j2
+@RestController
+@RequestMapping("/mandatory-questions")
+@Tag(name = "Mandatory questions", description = "API for handling mandatory questions")
+@RequiredArgsConstructor
+public class GrantMandatoryQuestionsController {
+
+    final private GrantMandatoryQuestionService grantMandatoryQuestionService;
+
+    @GetMapping("/does-scheme-have-completed-mandatory-questions/{schemeId}")
```

Comment:
This could be significantly shorter. The base REST resource we are accessing is `/mandatory-questions` so we already know that everything after that will be related to MQ's. With that in mind I'd suggest something a bit more RESTFUL might be `/mandatory-questions/scheme/{schemeId}/complete`